clang/lib/Format/Format.cpp

```diff
@@ -1199,6 +1199,15 @@ template <> struct DocumentListTraits<std::vector<FormatStyle>> {
     return Seq[Index];
   }
 };
+
+template <> struct ScalarEnumerationTraits<FormatStyle::GotoLabelIndentation> {
+  static void enumeration(IO &IO, FormatStyle::GotoLabelIndentation &Value) {
+    IO.enumCase(Value, "None", FormatStyle::GLI_None);
+    IO.enumCase(Value, "Indent", FormatStyle::GLI_Indent);
+    IO.enumCase(Value, "HalfIndent", FormatStyle::GLI_HalfIndent);
+  }
+};
+
```

Comment:
You are not supporting the old true/false case this would cause everyone else to break

Reply:
working on it

Reply:
Ok this should have been in addition to what you had before not instead of